_plugins/after_init.rb

```diff
@@ -0,0 +1,16 @@
+# Copy the specified documentation repos into the jekyll-tracked "doc" directory.
```

Comment:
@apojomovsky nit: consider a more descriptive file name, like `pulling_docs.rb` or something like that.